src/containers/card/reducer.js

```diff
@@ -0,0 +1,125 @@
+import { CLEAN_STATE,
+  GET_ALL_COMMENTS_IN_CARD, GET_ALL_LABELS_IN_CARD, GET_ALL_ASSIGNEES_IN_CARD, GET_RESPONSIBLE_FOR_CARD, GET_ONE_CARD,
+  ADD_COMMENT, ADD_LABEL, ADD_MEMBER, ADD_RESPONSIBLE,
+  UPDATE_DUE_DATE, UPDATE_DESC } from './constants'
+
+const initialState = {
+  title: '',
+  desc: '',
+  rank: '',
+  listId: '',
+  cardResponsible: null,
+  dueComplete: null,
+  labels: [],
+  assignees: [],
+  comments: [],
+  isAddingComment: false,
+  isAddingMember: false,
+  isAddingLabel: false,
+  isAddingDueDate: false,
+  isAddingDesc: false,
+}
+
+export default (state = initialState, action) => {
+  switch (action.type) {
+    case `${CLEAN_STATE}`:
+      return initialState
+    case `${GET_ALL_COMMENTS_IN_CARD}_SUCCESS`:
+      return {
+        ...state,
+        comments: action.payload.data,
+      }
+    case `${GET_ALL_LABELS_IN_CARD}_SUCCESS`:
+      return {
+        ...state,
+        labels: action.payload.data,
+      }
+    case `${GET_ALL_ASSIGNEES_IN_CARD}_SUCCESS`:
+      return {
+        ...state,
+        assignees: action.payload.data,
+      }
+    case `${GET_RESPONSIBLE_FOR_CARD}_SUCCESS`:
+      return {
+        ...state,
+        responsible: action.payload.data,
+      }
+    case `${GET_ONE_CARD}_SUCCESS`:
+      console.log('action.payload.data')
+      console.log(action.payload.data)
+      const ourCard = {
+        ...state,
+        ...action.payload.data,
+      }
+      console.log(ourCard)
+      return ourCard
+    case `${ADD_COMMENT}_SUCCESS`:
+      return {
+        ...state,
+        data: state.data(action.payload.data),
```

Comment:
`data` property doesn't exist in this state, correct this and all other case reducers